examples/sum_views.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <beman/any_view/any_view.hpp>
+
+#include <iostream>
+#include <ranges>
+#include <vector>
+
+namespace bav = beman::any_view;
+using opt     = bav::any_view_options;
+
+template <class T>
+using proxy_any_view = bav::any_view<T, opt::input, T>;
+
+constexpr auto sum(proxy_any_view<proxy_any_view<int>> views) {
```

Comment:
```suggestion
constexpr auto sum(bav::any_view<int, opt::input, int> views) {
```
am i missing something -- that seems as short as the proxy?

wait that's wrapped around itself -- if we're going to do this we better explain it...

Reply:
I added some comments for this in the code, should we add more explanations?

Reply:
I guess I'd like to see the example just use any_view and not conflate it with a template that I don't understand (even after explanation unfortunately).  I thought the point here was to be similar to span, but able to handle sort all kinds of views.  So something like

```
auto sum( any_view<int, opt::input> v1, any_view<int, opt::input v2 )
{...}
```
which would allow v1 and v2 to be say vector<int> and array<int> respectively.

Reply:
I changed the examples, to something similar with what you mentioned. THanks for pointing out something more easy to follow.